validate.py

```diff
@@ -11,12 +11,12 @@
 # the regex module has the same API as re but it can check Unicode character properties using \p{}
 # as in Perl.
 #import re
-import regex as re
```

Comment:
Why move the line without also moving the comments above that are directly related to the line?

Reply:
You're right, my bad! I'll update it :)